Frontend/ui-customer/src/Helpers/Utils.purs

```diff
@@ -550,4 +556,12 @@ getVariantRideType variant =
                     "TAXI" -> getString NON_AC_TAXI
                     "SUV"  -> getString AC_SUV
                     _      -> getString AC_CAB
-    _          -> getString AC_CAB
\ No newline at end of file
+    _          -> getString AC_CAB
+
+getTitleConfig :: forall w. String -> {text :: String , color :: String}
+getTitleConfig vehicleVariant =
+  (case vehicleVariant of
+        "TAXI" -> {text : (getString NON_AC )<> " " <> (getString TAXI) , color : CommonColor.orange900 }
```

Comment:
create a subFuntion which will make the return value - 

```purescript
where
   mkReturnObj textVal colorVal = 
     { text: textVal
       color: colorVal
     }
``` 

and then use it like this -

```purescript
 "TAXI" -> mkReturnObj .... .....
```